quic/s2n-quic-transport/src/stream/stream_container.rs

```diff
@@ -145,10 +145,12 @@ struct InterestLists<S> {
     /// stream flow control window to increase
     waiting_for_stream_flow_control_credits:
         LinkedList<WaitingForStreamFlowControlCreditsAdapter<S>>,
+    counter: u8,
+    limit: u8,
```

Comment:
i would probably prefix this with `transmission_` so it's clear what they're used for